include/zephyr/sys/slist.h

```diff
@@ -420,6 +420,21 @@ Z_GENLIST_REMOVE(slist, snode)
 static inline bool sys_slist_find_and_remove(sys_slist_t *list,
 					     sys_snode_t *node);
 
+/**
+ * @brief Find if a node is already linked in a singly linked list
+ *
+ * This and other sys_slist_*() functions are not thread safe.
+ *
+ * @param list A pointer to the list to check
+ * @param node A pointer to the node to search in the list
+ * @param prev A pointer to the previous node (output parameter)
```

Comment:
Just a minor nit, otherwise, LGTM
```suggestion
 * @param[out] prev A pointer to the previous node
```